### PR TITLE
Implement uri parsing for remote feed entries

### PR DESCRIPTION
**Pull request by isimluk**

### CODE REVIEW

File: pkg/rolie/clone.go [hunk 1]
```diff
@@ -108,11 +110,17 @@ func (f *fetcher) getRemoteResourceRaw(URI string) (io.ReadCloser, error) {
...
 func (f *fetcher) storeLocally(URI string, content []byte) error {
-	filepath, err := f.filepath(URI)
+	path, err := f.filepath(URI)
 	if err != nil {
 		return err
 	}
-	return ioutil.WriteFile(filepath, content, 0777)
+
+	dirPath := filepath.Dir(path)
+	err = os.MkdirAll(dirPath, os.FileMode(0722))
```
Comment (redhatrises) [inline pkg/rolie/clone.go]: @isimluk file mode of `0722`?

Comment (isimluk) [reply pkg/rolie/clone.go]: Nice catch! THX

File: pkg/rolie/clone.go [hunk 2]
```diff
@@ -108,11 +110,17 @@ func (f *fetcher) getRemoteResourceRaw(URI string) (io.ReadCloser, error) {
...
 		return err
 	}
-	return ioutil.WriteFile(filepath, content, 0777)
+
+	dirPath := filepath.Dir(path)
+	err = os.MkdirAll(dirPath, os.FileMode(0755))
+	if err != nil {
+		return err
+	}
+	return ioutil.WriteFile(path, content, 0777)
```
Comment (redhatrises) [inline pkg/rolie/clone.go]: @isimluk I think that I missed this one too... should this be 0644?

Comment (isimluk) [reply pkg/rolie/clone.go]: Fixed! thx

**Pull request merged**